images/ami/rhel-82.yaml

```diff
@@ -1,15 +1,15 @@
 download_images: true
 
 packer:
-  ami_filter_name: "RHEL-8.2*-x86_64-*"
+  ami_filter_name: "RHEL-8.2.0*2020*-x86_64-*"
```

Comment:
I really don't know what the deeper implications of this are.... This should probably be communicated to all of our customers who are using RH 8.2

Reply:
Indeed that is my biggest worry as well. We are definitely not working based on the latest RHEL8.2 release. That is why I was hoping to get more buy-in from the other KIB stakeholders.